persistence-modules/hibernate5/src/test/java/com/baeldung/hibernate/findall/FindAllTest.java

```diff
@@ -0,0 +1,65 @@
+package com.baeldung.hibernate.findall;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.baeldung.hibernate.HibernateUtil;
+import com.baeldung.hibernate.pojo.Student;
+
+public class FindAllTest {
+
+    private Session session;
+    private Transaction transaction;
+    
+    private FindAll findAll;
+
+    @Before
+    public void setUp() throws IOException {
+
+        session = HibernateUtil.getSessionFactory()
+            .openSession();
```

Comment:
In this case and on line 33 you can safely keep these fluent api calls on the same line. We only break up these calls when they are following out of the predefined page area, or if needed in our articles

Reply:
Same thing for `getResultList()` in `FindAll`

Reply:
Baeldung formatter breaks it down on save. I'll do it here, we need to update that formatter ...